test_runner/src/main/kotlin/ftl/run/TestRunner.kt

```diff
@@ -278,6 +308,18 @@ object TestRunner {
         ReportManager.generate(matrixMap)
     }
 
+    // used to cancel and update results from an async run
+    suspend fun cancelLastRun(config: IArgs) {
+        val matrixMap = lastMatrices()
+
+        cancelMatrices(matrixMap, config)
+        pollMatrices(matrixMap, config)
```

Comment:
if we're cancelling then I don't think we need to poll, fetch artifacts, or generate reports.